Documentation/project-docs/glossary.md

```diff
@@ -87,3 +87,62 @@ Mono has implementations (though not necessarily complete) of WinForms, ASP.NET,
 ### Xamarin
 
 Xamarin is a commercial offering for building mobile applications targeting Android, iOS and Mac OS X Store. It's based on Mono, and on iOS and Android surfaces a different API profile, called the mobile profile. The subsetting was necessary to reduce the footprint, both by shipping smaller versions of the system libraries as well as making them more linker friendly. While Mono runs on Mac OS X without Xamarin, their linker is required make the app package for the Mac App Store.  Xamarin ships a full static compiler on iOS, as the platform does not support dynamic code generation.
+
+
+## Frameworks
+
+### LINQ
+
+### ADO.NET
+
+### Managed Extensibility Framework
+
+**Also referred to as**: MEF
+
+### Windows Communication Foundation
+
+**Also referred to as**: WCF, Indigo
+
+### Entity Framework
+
+**Also referred to as**: EF
+
+### Entity Framework Core
+
+**Also referred to as**: EF Core
+
+### JSON.NET
+
+### Windows Forms
+
+**Also referred to as**: WinForms
+
+Windows Forms is an API provided by the .NET Framework (mostly in the `System.Windows.Forms` namespace) for creating desktop applications. Windows Forms provides an event-driven model for application development on top of the native loop-driven Win32 model. Mono has an implementation of Windows Forms, though it is not complete, since some parts of Windows Forms are tied to the Windows platform.
+
+Windows Forms is in maintenance mode currently. That means new features will generally not be added.
+
+### Windows Presentation Foundation
+
+**Also referred to as**: WPF, Avalon
+
+### DirectX
+
+**Also referred to as**: DX
+
+### XNA
+
+**Also referred to as**: XNA Framwork
+
+### MonoGame
+
+**Also referred to as**: MG, XNA Touch, Silver Sprite
```

Comment:
I'm not sure exactly what this is for, but I would prefer we don't suggest XNA Touch or Silver Sprite as other names for MonoGame.  Those are really old now... most people have never heard of them.  

They are legacy... no one should think Silver Sprite is an alternative name for MonoGame.